App/Logic/ExposureDetectionLogic.swift

```diff
@@ -325,26 +325,36 @@ extension Logic.ExposureDetection {
         return
       }
 
-      guard case .background = self.type else {
-        // Only trigger notifications for background tasks
-        return
-      }
-
       let title: String
       let message: String
       switch self.outcome {
       case .noDetectionNecessary:
+        let state = context.getState()
+        let enStatus = state.environment.exposureNotificationAuthorizationStatus
+        let latestProcessedChunk = state.exposureDetection.latestProcessedKeyChunkIndex
+
+        let lastDetectionString: String
+        if let lastDetectionDate = state.exposureDetection.lastDetectionDate {
+          let hoursSinceLastDetection = context.dependencies.now().timeIntervalSince(lastDetectionDate) / 3600
+          lastDetectionString = "\(String(format: "%.1f", hoursSinceLastDetection)) hours ago"
+        } else {
+          lastDetectionString = "never"
+        }
+
         title = "Skipped"
-        message = "Background task was skipped"
-      case .fullDetection, .partialDetection:
+        message = "\(Date().fullDateWithMillisString)\nExposure detection was skipped (enStatus: \(enStatus), lastDetection: \(lastDetectionString), lastChunk: \(latestProcessedChunk.map { String($0) } ?? "none"))"
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 345;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 214 characters (line_length)